book/src/getting-started/3-qml-gui.md

```diff
@@ -33,4 +33,16 @@ It is again important to emphasize here that `MyObject` is just another QObject
 The only difference being that any invokable functions that are defined are defined in Rust, instead of C++.
 For QML, this doesn't make a difference though.
 
-But enough of that, let's get this project [building and running](./5-cmake-integration.md).
+# Qt resources
+
+To include the `main.qml` file inside the application, use the [Qt resource system](https://doc.qt.io/qt-6/resources.html) by listing it in a `qml.qrc` file in the `qml` folder:
```

Comment:
The Qt cmake API `qt_add_qml_module()` has a neat option `QML_FILES` where you can list all QML files and it will automatically generate the qrc files, so that the user doesn't have to create any qrc files manually. I wonder how hard it is to add this feature to cxx-qt as well

Maybe even add all QML files automatically to a qrc file (without requiring the user to list them anywhere), or would there be any downside to doing it fully automatically?

Reply:
Right, it's a minor thing, so let's consider it for a PR later (if ever)